### PR TITLE
mkosi: Add a swap partition

### PR DESCRIPTION
**Pull request by daandemeyer**
systemd-oomd keeps complaining about the lack of swap partition, so
let's add one.

### CODE REVIEW

File: mkosi.presets/system/mkosi.extra/usr/lib/repart.d/15-swap.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Partition]
+Type=swap
+SizeMinBytes=1G
```
Comment (bluca) [inline mkosi.presets/system/mkosi.extra/usr/lib/repart.d/15-swap.conf]: A bit large for swap for these small images, maybe 100M?

**Review by bluca — APPROVED** [review]

**Review by keszybz — APPROVED** [review]
LGTM, I guess. We could pull in `zram-generator`, but it's probably not available everywhere, so the solution with the explicit swap partition is OK.

**Comment by DaanDeMeyer** [comment]
@mrc0mmand Do we have mkswap on CentOS CI?

**Comment by bluca** [comment]
```
18:12:04 [  557.726133] testsuite-58.sh[73]: Formatting future partition 1.
18:12:04 [  557.726194] testsuite-58.sh[73]: mkswap binary not available.
```

@mrc0mmand can this be added? I'll add it for Ubuntu

**Comment by mrc0mmand** [comment]
> ```
> 18:12:04 [  557.726133] testsuite-58.sh[73]: Formatting future partition 1.
> 18:12:04 [  557.726194] testsuite-58.sh[73]: mkswap binary not available.
> ```
> 
> @mrc0mmand can this be added? I'll add it for Ubuntu

Huh, that's strange, I though mkswap is part of util-linux which we already depend on.

**Comment by DaanDeMeyer** [comment]
It's an /sbin thing, does that affect it at all?

**Comment by mrc0mmand** [comment]
> It's an /sbin thing, does that affect it at all?

What's more strange is that TEST-55 also uses mkswap and it seems to work there without issues. I'll spin up a CI VM to see what's going on.

**Comment by mrc0mmand** [comment]
Ahhh, the code block where you added `image_install mkswap` is guarded by `if ! get_bool "${TEST_NO_QEMU:=}"; then` and the C8S job runs with `TEST_NO_QEMU=1`. So just move it out of the block and everything should work again.

**Pull request merged**